src/sentry/uptime/models.py

```diff
@@ -156,6 +158,9 @@ class ProjectUptimeSubscription(DefaultFieldsModelExisting):
         "sentry.Environment", db_index=True, db_constraint=False, null=True
     )
     uptime_subscription = FlexibleForeignKey("uptime.UptimeSubscription", on_delete=models.PROTECT)
+    status = BoundedPositiveBigIntegerField(
+        default=ObjectStatus.ACTIVE, choices=ObjectStatus.as_choices()
```

Comment:
`db_default`

Reply:
Why is that different from here? https://github.com/getsentry/sentry/blob/225c4d4c80d0545ab79271e2e2e0582147b5dc01/src/sentry/monitors/models.py#L233-L235

Reply:
When adding a new column to an existing table you need to use db_default, so that the default remains in place during the deploy